packages/fxa-content-server/app/styles/_breakpoints.scss

```diff
@@ -5,9 +5,9 @@ $breakpoints: (
     '(max-width: 520px), (max-height: 640px), (orientation: landscape) and (min-width: 641px) and (max-height: 640px)',
   big: '(min-width: 521px) and (min-height: 641px)',
   balloonSmall: '(max-width: 960px)',
-  balloonBig: '(min-width: 960px)',
+  balloonBig: '(min-width: 961px)',
```

Comment:
not explicitly necessary while the balloon is disabled, but when the screen was exactly 960px (as the TV is) both small and big selectors were triggered